proxyui/src/main/webapp/WEB-INF/views/edit.jsp

```diff
@@ -266,1660 +466,1426 @@
-                    ajax: {
-                        url: '<c:url value="/api/servergroup"/>' + '?profileId=${profile_id}',
-                        dataType: "json",
-                        data: function (term, page) {
-                            return {search: term};
-                        },
-                        results: function(data){
-                            var myResults = [];
-                            myResults.push({id: 0, text: "Default"});
-                            jQuery.each(data.servergroups, function(index, value){
-                                myResults.push({
-                                    id: value.id,
-                                    text: value.name
-                                });
+                    results: function(data){
+                        var myResults = [];
+                        myResults.push({id: 0, text: "Default"});
+                        $.each(data.servergroups, function(index, value){
+                            myResults.push({
+                                id: value.id,
+                                text: value.name
                             });
-                            return {
-                                results: myResults
-                            };
-                        }
-                    },
-                    createSearchChoice: function(term, data) {
-                        if ($(data).filter(function() {
-                            return this.text.localeCompare(term)===0;
-                        }).length===0) {
-                            return {id:term, text:term, isNew:true};
-                        }
-                    },
-                    formatResult: function(term) {
-                        if (term.isNew) {
-                            return 'create "' + term.text + '"';
-                        } else {
-                            return term.text;
-                        }
-                    }
-                });
-                $("#serverGroupSelection").on("change", function(e) {
-                    if(e.added.isNew) {
-                        $.ajax({
-                            type: "POST",
-                            url: '<c:url value="/api/servergroup" />',
-                            data: ({name : e.added.id, profileId: "${profile_id}"}),
-                            success: function(data){
-                                setActiveServerGroup(data.id);
-                                reloadGrid("#serverGroupList");
-                            }
                         });
-                    } else {
-                        setActiveServerGroup(e.added.id);
+                        return {
+                            results: myResults
+                        };
                     }
-                });
-                populateGroups();
-
-            });
-            jQuery("#packages").jqGrid('navGrid','#packages',{
-                edit:false,
-         
```

Comment:
Not needed anymore. this was functionality to make sure the user changed a manually-inserted placeholder.

Since we use now HTML `placeholder` attribute, these checks are no longer necessary.